### PR TITLE
Add FannyWidget 1.0.1 as cask

### PR DESCRIPTION
**Pull request by hwine**
### Checklist
- [X] The commit message includes the cask’s name and version.
- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` left no offenses.

Additionally, when **adding a new cask**:
- [X] Checked there are no open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [X] Checked there are no closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [X] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [X] `brew cask install {{cask_file}}` worked successfully.
- [X] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/fanny.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'fanny' do
+  version '1.0.1'
```
Comment (vitorgalvao) [inline Casks/fanny.rb]: Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/f25b6babcd398abf48e33af3d887b2d00de1d661/Casks/kibako.rb).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/fanny.rb (hunk 1 above)
Comment (hwine) [inline Casks/fanny.rb]: However, the 'app' stanza does have to reference the specific version. I'm not familiar enough with ruby to know how to either reference the version in the app stanza, or put a wildcard there.

File: Casks/fanny.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  sha256 'faeea76bca6a599be99457a0ecbb9ff406f30f893c2be81d9d93208f485546d3'
+
+  url 'http://fannywidget.com/FannyWidget.zip'
+  name 'FannyWidget'
+  homepage 'http://fannywidget.com/'
+  license :oss
+
+  app 'Fanny 1.0.1/fanny.app'
```
Comment (vitorgalvao) [inline Casks/fanny.rb]: But here, make it `app "Fanny #{version}/fanny.app"`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/fanny.rb (hunk 2 above)
Comment (hwine) [inline Casks/fanny.rb]: Ah, double quotes AND '#' -- thanks!

Comment (vitorgalvao) [inline Casks/fanny.rb]: And we also have [other niceties](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/version.md#version-methods).

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**